<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntropyCalculateClass</class>
 <widget class="QWidget" name="EntropyCalculateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EntropyCalculate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="DSGUI::DSLineEdit" name="m_editOfficialName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="DSGUI::DSPushButton" name="m_btnImport">
       <property name="text">
        <string>导入</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="DSGUI::DSTableView" name="m_listViewElement"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="DSGUI::DSPushButton" name="m_btnNext">
       <property name="text">
        <string>下一页</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="DSGUI::DSPushButton" name="m_btnDataStandardization">
       <property name="text">
        <string>数据标准化</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DSGUI::DSTableView</class>
   <extends>QTableView</extends>
   <header location="global">TableView/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>DSGUI::DSPushButton</class>
   <extends>QPushButton</extends>
   <header>PushButton/PushButton.h</header>
  </customwidget>
  <customwidget>
   <class>DSGUI::DSLineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit/lineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="EntropyCalculate.qrc"/>
 </resources>
 <connections/>
</ui>
